test/message/timeout_throw.chakracore.out

```diff
@@ -1,4 +1,5 @@
 ReferenceError: 'undefined_reference_error_maker' is undefined
    at Anonymous function (*test*message*timeout_throw.js:*:*)
+   at ontimeout (timers.js:*:*)
```

Comment:
Seems like we are now matching V8 but would be good to understand why was ontimeout function not in call stack before.